### PR TITLE
Feature/configurator interior

### PR DESCRIPTION
**Pull request by Filip-A25**

### CODE REVIEW

File: src/modules/configurator/hooks/useConfigProperty.tsx [hunk 1]
```diff
@@ -66,5 +69,25 @@ export function useConfigProperty({
...
+      case "color":
+        return "Paint color";
+      case "wheels":
+        return "Wheels";
+      case "interior_variants":
+        return "Color";
+      default:
+        return;
+    }
+  };
```
Comment (mrkaza) [inline src/modules/configurator/hooks/useConfigProperty.tsx]: This function could be moved to utilities, and just pass it the `propertyName`

File: src/modules/configurator/state/paginationState.ts [hunk 2]
```diff
@@ -13,9 +13,24 @@ export const pageState = atom<Page[]>({
...
+
+export const activePageState = selector<Page>({
+  key: "configurator.activePageState",
+  get: ({ get }) => {
+    const pages = get(pageState);
+    const activePage = pages.find(page => page.isActive);
+
+    if (!activePage) {
+      throw new Error("No active page was found.");
+    }
```
Comment (mrkaza) [inline src/modules/configurator/state/paginationState.ts]: You shouldn't throw error inside the selector, if it's not async one. You can just return undefined if there is no page

**Review by mrkaza — APPROVED** [review]
Lgtm. Check few comments

File: src/modules/configurator/hooks/useConfigProperty.tsx (hunk 1 above)
Comment (Filip-A25) [reply src/modules/configurator/hooks/useConfigProperty.tsx]: Done.

File: src/modules/configurator/state/paginationState.ts (hunk 2 above)
Comment (Filip-A25) [reply src/modules/configurator/state/paginationState.ts]: Removed throw error, returned undefined.

**Pull request merged**